samples/highcharts/demo/packed-bubble-dashboard/demo.details

```diff
@@ -0,0 +1,11 @@
+---
+name: Packed bubble dashboard
```

Comment:
I wonder if we should avoid the term dashboard here, since it's not a Dashboard? 🤔

Reply:
Only thing I am unsure of is what to call it instead, @goransle ? 😬 
"task-overview", "todo", "project-plan", "project", "work-chart" idk 🤣